pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -251,379 +150,103 @@ def test_file_decriptors(self):
+            pulp_rpm.app.models.repository.RpmRepository, pulp_rpm.app.models.repository.RpmRemote
         """
-        repo = self.client.post(RPM_REPO_PATH, gen_repo())
-        self.addCleanup(self.client.delete, repo['pulp_href'])
-
-        # Create a remote with the unsigned RPM fixture url.
-        body = gen_rpm_remote(url=RPM_UNSIGNED_FIXTURE_URL)
-        remote = self.client.post(RPM_REMOTE_PATH, body)
-        self.addCleanup(self.client.delete, remote['pulp_href'])
-
-        # Sync the repository.
-        self.assertEqual(repo["latest_version_href"], f"{repo['pulp_href']}versions/0/")
-        sync(self.cfg, remote, repo)
-        repo = self.client.get(repo['pulp_href'])
-        self.assertDictEqual(get_content_summary(repo), RPM_FIXTURE_SUMMARY)
-
-        # Save a copy of the original packages.
-        original_packages = {
-            (
-                content['name'],
-                content['epoch'],
-                content['version'],
-                content['release'],
-                content['arch'],
-            ): content
-            for content in get_content(repo)[RPM_PACKAGE_CONTENT_NAME]
-        }
-
-        # Create a remote with a different test fixture with the same NEVRA but
-        # different digests.
-        body = gen_rpm_remote(url=RPM_SIGNED_FIXTURE_URL)
-        remote = self.client.post(RPM_REMOTE_PATH, body)
-        self.addCleanup(self.client.delete, remote['pulp_href'])
-
-        # Sync the repository again.
-        sync(self.cfg, remote, repo)
-        repo = self.client.get(repo['pulp_href'])
-        self.assertDictEqual(get_content_summary(repo), RPM_FIXTURE_SUMMARY)
-
-        # In case of "duplicates" the most recent one is chosen, so the old
-        # package is removed from and the new one is added to a repo version.
-        self.assertEqual(
-            len(get_added_content(repo)[RPM_PACKAGE_CONTENT_NAME]),
-            RPM_PACKAGE_COUNT,
-            get_added_content(repo)[RPM_PACKAGE_CONTENT_NAME],
-        )
-        self.assertEqual(
-            len(get_removed_content(repo)[RPM_PACKAGE_CONTENT_NAME]),
-            RPM_PACKAGE_COUNT,
-            get_removed_content(repo)[RPM_PACKAGE_CONTENT_NAME],
-        )
-
-        # Test that the packages have been modified.
-        mutated_packages = {
-            (
-                content['name'],
-                content['epoch'],
-                content['version'],
-                content['release'],
-                content['arch'],
-            ): content
-            for content in get_content(repo)[RPM_PACKAGE_CONTENT_NAME]
-        }
-
-        for nevra in original_packages:
-            with self.subTest(pkg=nevra):
-                self.assertNotEqual(
-                    original_packages[nevra]['pkgId'],
-                    mutated_packages[nevra]['pkgId'],
-                    original_packages[nevra]['pkgId'],
-                )
-
-
-class EPELSyncTestCase(unittest.TestCase):
```

Comment:
we need to convert this test case also

Reply:
this test case is 'test_sync_epel_repo' method in 'BasicSyncTestCase' class